wrapper.go

```diff
@@ -72,6 +72,14 @@ func (w *Wrapper) PutBitlist(bb []byte, maxSize uint64) {
 }
 
 func (w *Wrapper) appendBytesAsNodes(b []byte) {
+	// if byte list is empty, fill with zeros
+	if len(b) == 0 {
+		b = append(b, zeroBytes[:32]...)
+	}
+	// if byte list isn't filled with 32-bytes padded, pad
+	if rest := len(b) % 32; rest != 0 {
+		b = append(b, zeroBytes[:32-rest]...)
+	}
```

Comment:
Similarly to https://github.com/ferranbt/fastssz/pull/120/files#r1097354675, we need to check if zero bytes need to added as padding to the next 32 byte increment.